version/version.go

```diff
@@ -31,9 +35,10 @@ var (
 func String() string {
 	return fmt.Sprintf(`-------------------------------------------------------------------------------
 NGINX Ingress controller
-  Release:    %v
-  Build:      %v
-  Repository: %v
+  Release:       %v
+  Build:         %v
+  Repository:    %v
+  %v
 -------------------------------------------------------------------------------
-`, RELEASE, COMMIT, REPO)
+`, RELEASE, COMMIT, REPO, nginx.Version())
```

Comment:
Please move this change into a separate PR too.

Reply:
done